src/datachain/lib/func/__init__.py

```diff
@@ -1,14 +1,20 @@
 from .aggregate import any_value, avg, collect, concat, count, max, min, sum
-from .func import Func
+from .func import Func, Window
+from .window import dense_rank, first, rank, row_number
 
 __all__ = [
     "Func",
+    "Window",
```

Comment:
it should be `window` (all lower case). Even in IBIS, I think it's lower case?:

```
w = ibis.window(group_by="one", preceding=5, following=5)
expr = t.mutate(group_demeaned=t.two - t.two.mean().over(w))
ibis.to_sql(expr)
```
in datachain upper case looks weird

(event though it might be a class underneath - it doesn't matter in this case I think)